scripts/extensions/broadcasting/src/interfaces.ts

```diff
@@ -61,6 +61,7 @@ export interface IRundownItemBase {
         }
     };
     subitems?: Array<ISubitem>;
+    _id?: string;
```

Comment:
that's defined in the `IBaseRestApiResponse` so should be available on `IRundownItem` type

Reply:
i know that... but i have error in TS if i dont add in `IRundownItem` too